app/helpers/Emails/EmailMacros.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Helpers\Emails;
+
+use Latte;
+use Latte\MacroNode;
+use Latte\Macros\MacroSet;
+
+class EmailSubject {
+
+  /**
+   * Holds last scraped subject from email template.
+   * @var string|null
+   */
+  public static $subject;
+
+  public static function clear() {
+    self::$subject = null;
+  }
+}
+
+class EmailMacros extends MacroSet {
+
+  public static function install(Latte\Compiler $compiler): MacroSet {
+    $set = new static($compiler);
+    $set->addMacro('emailSubject', [$set, 'macroEmailSubject'], [$set, 'macroEndEmailSubject']);
+    return $set;
+  }
+
+  public function macroEmailSubject(MacroNode $node, Latte\PhpWriter $writer) {
+    return 'ob_start(function () {})';
```

Comment:
I did not gather, why is it necessary to pass on empty lambda function as a callback to ob_start().

Reply:
Descriptive comment added